src/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": true,
+    "lib": ["es2018", "dom"],
+    "jsx": "react",
+    "jsxFactory": "createElement",
+    "module": "commonjs",
+    "noEmit": true,
+    "strictNullChecks": true,
```

Comment:
Maybe we should consider the possibility to add `"strict": true`. It checks for more than just `strictNullChecks`
https://www.typescriptlang.org/docs/handbook/compiler-options.html

Maybe we could allow for `implicitAny`. I suggest this:
```
    "strict": true,
    "noImplicitAny": false,
```